server/api.js

```diff
@@ -27,8 +24,27 @@ router.get("/image", (req, res) => {
 	}
 });
 
-router.put("/image", (req, res) => {
-	res.json({ message: "Put image" });
-});
+router.put(
+	"/image",
+	uploadImage.single("image"), // our uploadImage middleware
+	(req, res) => {
+		/*
+           req.file = {
+             fieldname, originalname,
+             mimetype, size, bucket, key, location
+           }
+        */
```

Comment:
We could delete this commented out code

Reply:
Yes, thanks. I fixed in this commit: https://github.com/Innapoliakova/set/pull/50/commits/53b418d3a4cc031d76573d7a38a100c480404520